### PR TITLE
[TR]PIM-5207: split features for better distributed performances

### PR DESCRIPTION
**Pull request by solivier**
Features that are too long to execute are split in several files in order to take less than 10 mins to be executed on a worker.

### CODE REVIEW

File: features/mass-action/edit_common_attributes-01.feature [hunk 1]
```diff
@@ -0,0 +1,159 @@
+@javascript
```
Comment (juliensnz) [inline features/mass-action/edit_common_attributes-01.feature]: please use underscore (you did it for res/attribute/option/add_attribute_options_03.feature, res/attribute/option/add_attribute_options_02.feature) to be consistent

**Comment by pierallard** [comment]
Some files are named xxx_01.feature, some are named xxx-01.feature :)

**Comment by juliensnz** [comment]
GT after comment fix

**Comment by nidup** [comment]
please find real functional names and avoid 01, 02, 03, 04 etc

**Comment by jmleroux** [comment]
GTM after renaming

**Comment by grena** [comment]
GTM (all but the completeness thing :) )

**Pull request merged**